modular_zubbers/code/modules/bitrunning/disks.dm

```diff
@@ -0,0 +1,50 @@
+/obj/item/bitrunning_disk/prefs
+	name = "DeForest biological simulation disk"
+	desc = "A disk containing the biological simulation data necessary to load custom characters into bitrunning domains."
+	icon = 'icons/obj/devices/circuitry_n_data.dmi'
+	base_icon_state = "datadisk"
+	icon_state = "datadisk0"
+
+	w_class = WEIGHT_CLASS_SMALL
+
+	var/datum/preferences/loaded_preference
+
+	var/include_loadout = FALSE
+
+/obj/item/bitrunning_disk/prefs/examine(mob/user)
+	. = ..()
+	if(!isnull(loaded_preference))
+		var/name = loaded_preference.read_preference(/datum/preference/name/real_name)
+		. += "It currently has the character [name] loaded, with loadouts [(include_loadout ? "enabled" : "disabled")]"
+		. += span_notice("Ctrl-Click to change loadout loading")
```

Comment:
```suggestion
		. += span_notice("Ctrl-Click to change loadout loading")
	else
		. += Is empty.
		. += span_notice("Use the disk on yourself to load a character")
```
Wording on that notice I wrote seems a little awkward

Reply:
Disks already come with a use in hand warning by default